### PR TITLE
[Enhancement] add audit log for export

### LINKED ISSUE #16565: Add audit log for export
## Enhancement
Current audit log for export is null
image

### PR DESCRIPTION
**Pull request by xiaoyong-z**
Signed-off-by: xyz <a997647204@gmail.com>

## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #16565

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
- [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
- [x] 2.5
- [ ] 2.4
- [ ] 2.3
- [ ] 2.2

### CODE REVIEW

**Review by meegoo — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 5 / 24 (20.83%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/load/ExportJob.java|0|2|00.00%|[422, 423]|
|:large_blue_circle:|com/starrocks/sql/analyzer/AstToStringBuilder.java|5|22|22.73%|[243, 256, 258, 259, 260, 262, 265, 266, 267, 270, 271, 272, 274, 275, 276, 277, 278]|

File: fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AstToStringBuilder.java [hunk 1]
```diff
@@ -239,6 +240,42 @@ public Object apply(DataDescription dataDescription) {
...
+                Joiner.on(",").appendTo(sb, stmt.getColumnNames()).append(")");
+            }
+            sb.append(" TO ");
+            sb.append("\"" + stmt.getPath() +  "\" ");
+            if (stmt.getProperties() != null && !stmt.getProperties().isEmpty()) {
+                sb.append("PROPERTIES (");
+                sb.append(new PrintableMap<String, String>(stmt.getProperties(), "=", true, false));
+                sb.append(")");
+            }
+            sb.append("WITH BROKER");
```
Comment (rickif) [inline fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AstToStringBuilder.java:270]:
```suggestion
            sb.append("WITH BROKER ");
```

**Review by chaoyli — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

File: fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AstToStringBuilder.java [hunk 2]
```diff
@@ -239,6 +240,42 @@ public Object apply(DataDescription dataDescription) {
...
+            if (stmt.getBrokerDesc() != null) {
+                sb.append(stmt.getBrokerDesc().getName()).append("' (");
```
Comment (imay) [inline fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AstToStringBuilder.java:272]: name may be null?

**Review by trueeyu — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 5 / 26 (19.23%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/load/ExportJob.java|0|2|00.00%|[422, 423]|
|:large_blue_circle:|com/starrocks/sql/analyzer/AstToStringBuilder.java|5|24|20.83%|[243, 256, 258, 259, 260, 262, 265, 266, 267, 270, 271, 272, 274, 275, 276, 277, 278, 279, 281]|

**Review by chaoyli — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by xiaoyong-z** [comment]
@megrify backport branch-2.5

**Comment by xiaoyong-z** [comment]
@mergify backport branch-2.5

**Comment by xiaoyong-z** [comment]
@mergify backport branch-2.4

**Pull request merged**